agentai/conversation.py

```diff
@@ -51,7 +51,7 @@ def get_history(self) -> List[Message]:
 
         total_tokens = 0
         # Iterate 2 at a time to avoid cutting in between a (prompt, response) pair
-        for i in range(len(self.history) -1, -1, -2):
+        for i in range(len(self.history) - 1, -1, -2):
```

Comment:
Is this a typo? Why add a space here?

Reply:
This was done by black formatter. Will check